Project3.md

```diff
@@ -84,6 +83,34 @@ will provide decent default styling. However, if you want, you can
 think about how to improve the design or make it responsive for
 smaller screens.
 
+User Stories and Unit Testing
+-----------------------------
+
+Before starting to write any code for Project 3, start with developing a set of user stories. 
+Each feature that you code from this project onwards should correspond to a user story. Each user story should have a documented set of unit tests that validate the user story is correctly implemented (acceptance criteria). For this project, create at least 5 user stories, but no more than 20. Turn in your user stories as part of your project.
+
+### Unit Testing ###
+
+[Django's unit tests](https://docs.djangoproject.com/en/1.10/topics/testing/overview/) use the Python unit testing framework ```unitest```. To run your unit tests, execute ```python manage.py test``` from the command line. By default, this will run all files in your project named ```test*.py``` where * is a wildcard (ex. test_users.py).
+
+Here are some examples of unit tests for your Django views:
+
+    from django.test import TestCase
+    from myapp.models import Order
+    from myapp.models import User
+    
+    class OrderDetailTestCase(TestCase):
+      def setUp(self):     #setUp method is called before each test in this class
+         
```

Comment:
A complete unit test is what I tried to accomplish. Do you mean one with setup included or was there something else specific that I missed?